src/nonce.iced

```diff
@@ -0,0 +1,18 @@
+nonceBytes = 24
```

Comment:
This one is definitionally the same as `crypto_secretbox_NONCEBYTES`. We could use that name if you want.